pkg/scheduler/scheduler.go

```diff
@@ -285,36 +286,49 @@ func findFlavorForResource(
 	wUsed map[string]int64,
 	resourceFlavors map[string]*kueue.ResourceFlavor,
 	cq *cache.ClusterQueue,
-	spec *corev1.PodSpec) (string, int64) {
+	spec *corev1.PodSpec) (string, int64, []error) {
+	var errs []error
+
+	if _, exists := cq.RequestableResources[name]; !exists {
+		errs = append(errs, fmt.Errorf("resource %s not found", name))
```

Comment:
"resource unavailable in ClusterQueue"

There is no need to include the resource name, as it's already part of the message in `assignFlavors`.